Framework/Core/src/HistogramRegistry.cxx

```diff
@@ -114,6 +114,11 @@ HistPtr HistogramRegistry::add(char const* const name, char const* const title,
   return insert({name, title, {histType, axes}, callSumw2});
 }
 
+HistPtr HistogramRegistry::add(const std::string_view& name, char const* const title, HistType histType, const std::vector<AxisSpec>& axes, bool callSumw2)
+{
+  return insert({name.data(), title, {histType, axes}, callSumw2});
```

Comment:
a string_view is not necessarily null terminated. Is this taken into account?

Reply:
Indeed, I moved to std:: string